opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -123,7 +125,7 @@ namespace Opm {
 
 
         const UnitSystem& m_deckUnitSystem;
-        const ParseContext& m_parseContext;
+        const ParseContext m_parseContext;
```

Comment:
I agree that this is certainly not gonna affect performance in any measurable way. I think it is a bad habit which should not creep in unless there is a specific reason because sometimes it _does_ matter...

> - The copy is only done once because of moves.

that's actually only true if the `ParseContext` contains a lot of data allocated on the stack (i.e., copying it is probably a relatively expensive enterprise contradicting you first statement). If the compiler could inline the constructor, it *might* be able to do some copy-elision tricks, but with the current design (split sources and headers) that's very unlikely IMO unless LTO is explicitly enabled. (it is currently not enabled by default because it caused some obscure problems in the past.)